DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
 
+## TruexAdRenderer tvOS API
 
-### Handling Ad Elimination
+This is an outline of public `TruexAdRenderer` methods and `TruexAdRendererDelegate` methods (referred to as events).
 
-Skipping video ads is completely the responsibility of the app code. The Uplynk Preplay API should provide enough information for the app to determine where the current pod end-point is, and, when appropriate, should fast-forward directly to this point when resuming playback.
 
+### Adding `TruexAdRenderer` to Your Project
 
-## TruexAdRenderer tvOS API
+The easiest way to add `TruexAdRenderer` is via CocoaPods. The renderer will be maintained and distributed on Github.
 
-This is an outline of public `TruexAdRenderer` methods and delegate methods (referred to as events).
+1. Add the following Podspec repositories to your Podfile:
+    ```ruby
+    source 'https://github.com/CocoaPods/Specs.git'
+    source 'https://github.com/socialvibe/cocoapod-specs.git'
+    source 'https://innovid-cocoapods:3bbe7445d58951a8f89c5e54c0ebfdd17039589e@github.com/Innovid/cocoapods-spec.git'
+    source 'https://github.com/YOU-i-Labs/YouiTVAdRenderer-CocoaPod.git'
+    ```
+1. Add the TruexAdRenderer pod to your target:
+    ```ruby
+    target 'MyApp' do
+        use_frameworks!
+        pod 'TruexAdRenderer', '~> 3.6'
+        # [...]
+    end
+    ```
 
+**Note:** `TruexAdRenderer` and some of its dependencies are delivered as frameworks -- not as static libraries. As such, please include the `use_frameworks!` line in your target definition.
 
 ### TruexAdRenderer Methods
 
-When referring to data returned via Uplynk Preplay, I will reference this example JSON response in order to clearly explain which piece of data is required:
+#### `initWithUrl`
 
-```
-    https://gist.github.com/jalbini/79a6fcd4e12049b6f5ee1d883d372b81
+```objective-c
+    - (_Nullable id)initWithUrl:(NSString* _Nullable)creativeUrl adParameters:(NSDictionary* _Nonnull)adParameters slotType:(NSString* _Nonnull)slotType;
 ```
 
-I will refer to entire object as `response`, and will point to specific pieces of data using standard JSON notation.
+This method should be called by the app code in order to initialize the `TruexAdRenderer`. The renderer will parse out the `creativeURL`, `adParameters` and `slotType` passed to it and make a request to the true[X] ad server to see what ads are available.
 
+You may initialize `TruexAdRenderer` early (a few seconds before the next pod even starts) in order to give it extra time to make the ad request. The renderer will output an [`onFetchAdComplete`](#onfetchadcomplete) event at completion of this ad request. This event can be used to facilitate the implementation of a timeout or loading indicator, and when to make the call to `start`.
```

Comment:
Oh cool, we do mention this here! Maybe move some of this to "When to show true[X]" ?